l10n-pocket/en/pocket/tos.ftl

```diff
@@ -0,0 +1,128 @@
+pocket-tos-there-is-no-judge = There is no judge or jury in arbitration, and court review of an arbitration award is limited. an arbitrator, however, may award on an individual basis the same damages and relief as a court (including, without limitation, injunctive and declaratory relief or statutory damages), and must follow these terms of service as a court would. arbitration is more informal than a lawsuit in court, and allows for more limited discovery than in court. the arbitrator will decide all issues, including, without limitation, issues relating to the scope and enforceability of this arbitration provision.
+# Variables:
+#   $pocket_tos_mozilla_address: (String) President, Mozilla Corporation/Pocket Business Unit, 2 Harrison Street, Suite 175, San Francisco, CA 94105
+#   $jams_abbr: (String) JAMS
+#   $jams_url: (String) https://www.jamsadr.com
+#   $adr_url: (String) www.adr.org
+pocket-tos-to-initiate = To initiate an arbitration proceeding, you must send a letter requesting arbitration and describing your claim to: { $pocket_tos_mozilla_address}. Unless you and we agree otherwise, any arbitration will take place in the county (or parish) where you live. The arbitration will be conducted by  in accordance with that organization’s "Streamlined Arbitration Rules & Procedures" if { $jams_abbr } has an office in the county (or parish) where the arbitration takes place. Otherwise, the arbitration will be conducted by a neutral arbitrator in accordance with the American Arbitration Association (AAA) rules, including the AAA’s Supplementary Procedures for Consumer-Related Disputes. { $jams_abbr } rules and other information are available at { $jams_url } or by calling 800.352.5267. The AAA’s rules and other information are available at { $adr_url } or by calling 1-800-778-7879. Payment of all filing, administration and arbitrator fees will be governed by the applicable { $jams_abbr } or AAA rules. We will promptly reimburse you for your payment of the filing fees, unless your claim is for greater than US$10,000, in which event you will be responsible for filing fees. If your claim is for US$10,000 or less, we agree that you may choose whether the arbitration will be conducted solely on the basis of documents submitted to the arbitrator, through a telephonic hearing, or by an in-person hearing as established by the applicable { $jams_abbr } or AAA rules. The arbitrator shall have the discretion to award reasonable attorneys’ fees and costs (including the cost of the arbitration and of experts) to the prevailing party, but we will only seek such fees if the arbitrator determines that your claim is frivolous.
```

Comment:
**Nit (blocking):** You forgot a word in the string here. It should be `The arbitration will be conducted by JAMS in accordance with..`
```suggestion
pocket-tos-to-initiate = To initiate an arbitration proceeding, you must send a letter requesting arbitration and describing your claim to: { $pocket_tos_mozilla_address}. Unless you and we agree otherwise, any arbitration will take place in the county (or parish) where you live. The arbitration will be conducted by JAMS in accordance with that organization’s "Streamlined Arbitration Rules & Procedures" if { $jams_abbr } has an office in the county (or parish) where the arbitration takes place. Otherwise, the arbitration will be conducted by a neutral arbitrator in accordance with the American Arbitration Association (AAA) rules, including the AAA’s Supplementary Procedures for Consumer-Related Disputes. { $jams_abbr } rules and other information are available at { $jams_url } or by calling 800.352.5267. The AAA’s rules and other information are available at { $adr_url } or by calling 1-800-778-7879. Payment of all filing, administration and arbitrator fees will be governed by the applicable { $jams_abbr } or AAA rules. We will promptly reimburse you for your payment of the filing fees, unless your claim is for greater than US$10,000, in which event you will be responsible for filing fees. If your claim is for US$10,000 or less, we agree that you may choose whether the arbitration will be conducted solely on the basis of documents submitted to the arbitrator, through a telephonic hearing, or by an in-person hearing as established by the applicable { $jams_abbr } or AAA rules. The arbitrator shall have the discretion to award reasonable attorneys’ fees and costs (including the cost of the arbitration and of experts) to the prevailing party, but we will only seek such fees if the arbitrator determines that your claim is frivolous.
```